docs/guide/data_analysis.ipynb

```diff
@@ -93,7 +93,7 @@
         "try:\n",
         "    import recirq\n",
         "except ImportError:\n",
-        "    !pip install git+https://github.com/quantumlib/ReCirq"
+        "    !pip install --quiet git+https://github.com/quantumlib/ReCirq"
```

Comment:
Getting an install error when I run in Colab: https://colab.research.google.com/github/mpharrigan/ReCirq/blob/2020-11-docs-1/docs/guide/data_collection.ipynb

```
ERROR: pytket 0.6.1 has requirement sympy~=1.5, but you'll have sympy 1.1.1 which is incompatible.
ERROR: pytket-cirq 0.5.0 has requirement cirq~=0.8.0, but you'll have cirq 0.9.1 which is incompatible.
```

The Colab environment might default to a different version than you need. Or maybe pin a version when pip installing?

Reply:
Yeah, it says `ERROR` but it should still work. The message is a result of overzealous pinning in cirq *and* pytket

Reply:
xref #61 and https://github.com/CQCL/pytket/issues/22 -- which I fixed but looks like we need to bump again. It doesn't help that pytket is closed source so they have to make the change